### PR TITLE
docs: release notes for v2.30

### PR DESCRIPTION
**Pull request by vladar**
## Description

Release notes for v2.30. Please add anything you would like to mention in this release.

[Rendered version](https://github.com/gatsbyjs/gatsby/blob/release-notes-2.30/docs/docs/reference/release-notes/v2.30/index.md)

### CODE REVIEW

File: docs/docs/reference/release-notes/v2.30/index.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+- [Query on Demand and Lazy Images: Generally available](#query-on-demand-and-lazy-images-generally-available) - improves `gatsby develop` bootup time
+- [Server Side Rendering (SSR) in development](#server-side-rendering-ssr-in-development) — helps you find and fix many build errors in development. We're starting a partial release of this feature to 5% of users.
+- [Image transformations up to 30% faster](#gatsby-plugin-sharp2120)
+- [gatsby-plugin-sass v3](#gatsby-plugin-sass300) - use the latest `sass-loader` and `Dart Sass` by default
+
+And several impactful updates in the new [`gatsby-plugin-image`](#gatsby-plugin-image050-beta) (beta):
+
+- [AVIF support](#avif-support) - enjoy next-gen image format
+- [Support remote static images](#support-static-image-urls-experimental) - download, transform and optimize remote images with a single line
+- [New defaults](#new-defaults) - less configuration for most common image setups
```
Comment (pieh) [inline docs/docs/reference/release-notes/v2.30/index.md:23]: I don't see "New defaults" section in content?

File: docs/docs/reference/release-notes/v2.30/index.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+---
+
+Welcome to `gatsby@2.30.0` release (January 2021 #1)
+
+Key highlights of this release:
+
+- [Query on Demand and Lazy Images: Generally available](#query-on-demand-and-lazy-images-generally-available) - improves `gatsby develop` bootup time
+- [Server Side Rendering (SSR) in development](#server-side-rendering-ssr-in-development) — helps you find and fix many build errors in development. We're starting a partial release of this feature to 5% of users.
+- [Image transformations up to 30% faster](#gatsby-plugin-sharp2120)
+- [gatsby-plugin-sass v3](#gatsby-plugin-sass300) - use the latest `sass-loader` and `Dart Sass` by default
```
Comment (pieh) [inline docs/docs/reference/release-notes/v2.30/index.md:17]: Nit: table of content order doesn't match content order (in content we have sass before sharp) ;)

File: docs/docs/reference/release-notes/v2.30/index.md (hunk 1 above)
Comment (vladar) [reply docs/docs/reference/release-notes/v2.30/index.md:23]: True, looks like Matt decided not to include this part

File: docs/docs/reference/release-notes/v2.30/index.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+Now we feel confident enough to publish it as generally available. You don't need to add flags or any environment
+variables to enable it. Just upgrade to `gatsby@2.30.0` (and `gatsby-plugin-sharp@2.12.0` for lazy images).
+
+In v2.29 we improved the UX around long-running queries by adding a loading indicator and message to the browser console (only in `gatsby develop`). If you want or need to de-activate this indicator, you can! For more details please see the [umbrella discussion](https://github.com/gatsbyjs/gatsby/discussions/27620).
+
+## Server Side Rendering (SSR) in development
+
+There are certain types of build errors that currently are only discoverable when doing a build. The most common is code that tries to access browser globals (like `window`) that don't exist in Node.js when SSRing your Gatsby site.
+
+This causes a lot of frustration as you might develop for days before running a build and it’s not fun to discover problems long after they were created. Then actually fixing the problems is painful as the feedback cycle is slow as you have to run a build after each code change.
```
Comment (pieh) [inline docs/docs/reference/release-notes/v2.30/index.md:63]: Do we want to mention that in dev-ssr mode `style-loader` is no longer used and that styling inconsistencies between develop and build _might_ be addressed? (I think there are some edge cases when it's not exactly the same due to us forcing single css chunk for prod, but for dev it's possibly to create more when doing lazy imports?)

/cc @KyleAMathews

File: docs/docs/reference/release-notes/v2.30/index.md (hunk 1 above)
Comment (vladar) [reply docs/docs/reference/release-notes/v2.30/index.md:23]: @ascorbic I was thinking about including a note about this PR: https://github.com/gatsbyjs/gatsby/pull/28662 there Will remove this "New defaults" section for now.

Comment (ascorbic) [reply docs/docs/reference/release-notes/v2.30/index.md]: Oh, I didn't realise that was in this release!

**Review by pieh — DISMISSED** [review]
🎉

**Review by KyleAMathews — APPROVED** [review]
:shipit:

**Review by KyleAMathews — APPROVED** [review]

**Pull request merged**